src/main/java/rs/teslaris/core/service/interfaces/document/EventService.java

```diff
@@ -23,7 +24,15 @@ public interface EventService {
     Boolean hasCommonUsage(Integer eventId);
 
     Page<EventIndex> searchEvents(List<String> tokens, Pageable pageable,
-                                  EventType eventType);
+                                  EventType eventType, Boolean returnOnlyNonSerialEvents);
 
     Page<EventIndex> searchEventsImport(List<String> names, String dateFrom, String dateTo);
+
+    List<EventsRelationDTO> readEventRelations(Integer eventId);
+
+    List<EventsRelationDTO> readSerialEventRelations(Integer serialEventId);
+
+    void addEventsRelation(EventsRelationDTO eventsRelationDTO);
+
+    void deleteEventRelation(Integer relationId);
```

Comment:
please make consistent names of these two methods (one is EventsRelation, the another is EventRelation)

Reply:
Done